tests/test_database.py

```diff
@@ -4954,7 +4954,12 @@ def test_floating_point_limits(self):
         # outside of high bound for Delta
         with self.assertRaises(cantools.database.EncodeError):
             message.encode({ **data_max, 'Delta': data_max['Delta']+Decimal('0.001') })
-
+    def test_bus_comment(self):
+        filename = 'tests/files/dbc/bus_comment.dbc'
+        db1 = cantools.database.load_file(filename)
+        self.assertEqual(db1.buses[0].comment, 'SpecialRelease')
+        db2 = cantools.database.load_string( db1.as_dbc_string() )
+        self.assertEqual( db2.buses[0].comment, 'SpecialRelease' )
```

Comment:
Remove line 4961 and 4962. Line 4959 and 4960 are enough.

Also, test dump functionality. Right now only load functionality is tested.